addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -61,175 +69,461 @@
+         * 14.033: Frequency, values -3.40282347e+38f..3.40282347e+38f Hz
+         * 14.077: Volume flux, values -3.40282347e+38f..3.40282347e+38f m³/s
+         * 14.032: Force, values -3.40282347e+38f..3.40282347e+38f N
+         * 14.076: Volume, values -3.40282347e+38f..3.40282347e+38f m³
+         * 14.031: Energy, values -3.40282347e+38f..3.40282347e+38f J
+         * 14.075: Torque, values -3.40282347e+38f..3.40282347e+38f Nm
+         * 14.030: Electromotive force, values -3.40282347e+38f..3.40282347e+38f V
+         * 14.074: Time, values -3.40282347e+38f..3.40282347e+38f s
+         * 14.073: Thermoelectric power, values -3.40282347e+38f..3.40282347e+38f V/K
+         * 14.072: Thermal conductivity, values -3.40282347e+38f..3.40282347e+38f W/m K⁻¹
+         * 14.009: Angular velocity, values -3.40282347e+38f..3.40282347e+38f rad/s
+         * 14.008: Momentum, values -3.40282347e+38f..3.40282347e+38f Js
+         * 14.007: Angle, values -3.40282347e+38f..3.40282347e+38f °
+         * 14.006: Angle, values -3.40282347e+38f..3.40282347e+38f rad
+         * 14.005: Amplitude, values -3.40282347e+38f..3.40282347e+38f
+         * 14.049: Magnetization, values -3.40282347e+38f..3.40282347e+38f A/m
+         * 14.004: Mol, values -3.40282347e+38f..3.40282347e+38f mol
+         * 14.048: Magnetic polarization, values -3.40282347e+38f..3.40282347e+38f T
+         * 14.003: Activity, values -3.40282347e+38f..3.40282347e+38f s⁻¹
+         * 14.047: Magnetic moment, values -3.40282347e+38f..3.40282347e+38f A m²
+         * 14.002: Activation energy, values -3.40282347e+38f..3.40282347e+38f J/mol
+         * 14.046: Magnetic flux density, values -3.40282347e+38f..3.40282347e+38f T
+         * 14.001: Acceleration, angular, values -3.40282347e+38f..3.40282347e+38f rad s⁻²
+         * 14.045: Magnetic flux, values -3.40282347e+38f..3.40282347e+38f Wb
+         * 14.000: Acceleration, values -3.40282347e+38f..3.40282347e+38f ms⁻²
+         * 14.044: Magnetic field strength, values -3.40282347e+38f..3.40282347e+38f A/m
+         * 14.043: Luminous intensity, values -3.40282347e+38f..3.40282347e+38f cd
+         * 14.042: Luminous flux, values -3.40282347e+38f..3.40282347e+38f lm
+         * 14.041: Luminance, values -3.40282347e+38f..3.40282347e+38f cd m⁻²
+         * 14.040: Quantity of Light, values -3.40282347e+38f..3.40282347e+38f J
+         */
+        dptMainTypeMap.put(14, DecimalType.class);
+        /** Exceptions Datapoint Types "4-Octet Float Value", Main number 14 */
+        // Example: dptTypeMap.put(DPTXlator4ByteFloat.DPT_ACCELERATION_ANGULAR.getID(), DecimalType.class);
+
+        /**
+         * MainType: 16
+         * 16.000: ASCII string, values
+         * 16.001: ISO-8859-1 string (Latin 1), values
+         */
+        dptMainTypeMap.put(16, StringType.class);
+        /** Exceptions Datapoint Types "String", Main number 16 */
         dptTypeMap.put(DPTXlatorString.DPT_STRING_8859_1.getID(), StringType.class);
```

Comment:
@lewie Ok, I was in fact looking at this line before, and here we have StringTYpe for the maintype, and also String for the exceptions. That's why I could not figure out what you were doing. So, does that mean that we do need those exception here in class 16?